warehouse/cli/sponsors.py

```diff
@@ -0,0 +1,592 @@
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=True,
+      one_time=False,
+      sidebar=False,
+    ),
+    dict(
+      name="DigiCert",
+      service="EV certificate",
+      url="https://www.digicert.com/",
+      image="digicert.png",
+      activity=[
+        "PyPI uses Digicert to secure communication and prove identity with an EV Certificate."
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=True,
+      one_time=False,
+      sidebar=False,
+    ),
+    dict(
+      name="StatusPage",
+      service="Status page",
+      url="https://statuspage.io",
+      image="statuspage.png",
+      activity=[
+        "The PyPI team uses StatusPage to communicate downtime, service degradation, or maintenance windows to end-users."
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=True,
+      one_time=False,
+      sidebar=False,
+    ),
+    dict(
+      name="Mozilla",
+      url="https://www.mozilla.org",
+      image="mozilla.png",
+      activity=[
+        "In late 2017 the Python Software Foundation was awarded a Mozilla Open Source Support (MOSS) award.",
+        "This award was used to move PyPI from its legacy codebase and deploy Warehouse - the new codebase powering PyPI."
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=False,
+      one_time=True,
+      sidebar=False,
+    ),
+    dict(
+      name="Open Technology Fund",
+      url="https://www.opentech.fund",
+      image="otf.png",
+      activity=[
+        "In 2019 the Python Software Foundation was awarded a contract through the OTF Core Infrastructure Fund.",
+        "This contract included an audit and improvements to the accessibility of PyPI, internationalization and translations for PyPI user interface, and security features including two-factor authentication and API tokens for uploads.",
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=False,
+      one_time=True,
+      sidebar=False,
+    ),
+]
+
+
+@warehouse.group()  # pragma: no branch
+def sponsors():
+    """
+    Manage operations on top of sponsors.
+    """
+
+
+@sponsors.command()
+@click.pass_obj
+def populate_db(config):
+    """
+    Sync the Warehouse database with initial sponsors list.
+    Once this command is executed once, you shouldn't need to run
+    it again.
+    """
+    from warehouse.db import Session
+    session = Session(bind=config.registry["sqlalchemy.engine"])
+
+    for data in SPONSORS_DICTS:
+        name = data['name']
+        sponsor = session.query(Sponsor).filter_by(name=name).one_or_none()
+        if sponsor:
+            print(f'Skipping {name} sponsor because it already exists.')
+            continue
+
+        params = data.copy()
+        img = params.pop('image')
+        params["color_logo_url"] = img
+        params["white_logo_url"] = f"white/{img}"
```

Comment:
@ewdurbin as [you've comment](https://github.com/pypa/warehouse/issues/9503#issuecomment-836912380), I'm adding the URLs directly, but I don't think we can totally rely on current URLs in Pypi. I mean, take a look at current Google's logo URL:

https://pypi.org/static/images/sponsors/color/google.62f52fb9.png

It has this hash `62f52fb9` suffix that I'm assuming it changes every deploy. So, how can we define the URLs here?

Also, what do you think about the directory structure I proposed: having the colored logo in a directory and all the white logos should be placed in the `white` children directory with the same file name.